task_1.py

```diff
@@ -1,2 +1,10 @@
 def cross(n):
-    pass
+    for i in range(n):
+        print(" " * n + n * "*"+ n * " ")
+        i=i+1
+        if i==n:
+            for i in range(n):
+                print(n*3*"*")
```

Comment:
W tym przypadku o wiele lepiej usunąć teog ifa i po prostu zastosować 3 pętlę `for`.